src/ncp/ncp_base.cpp

```diff
@@ -342,7 +342,7 @@ void NcpBase::HandleActiveScanResult(otActiveScanResult *result)
             SPINEL_PROTOCOL_TYPE_THREAD,
             flags,
             result->mNetworkName,
-            result->mExtPanId, sizeof(result->mExtPanId)
+            result->mExtPanId, Thread::Mac::Beacon::kExtPanIdSize
```

Comment:
Good catch!  One note: we're actively working to make ncp-related code only rely on the public OpenThread API.  Can we change this to `sizeof(otExtAddress)` instead?